### ISSUE TITLE
/var/log/rvd/rvd.log.0 is not a useful file

### ISSUE THREAD

**Issue by erikbor** [riboseinc/rvc#212]
Hi @jjr840430 
It also appears that log rotation doesn't work because `/var/log/rvd/rvd.log` does not end up in `/var/log/rvd/rvd.log.0` when rotated.

The `/var/log/rvd/rvd.log.0` always has a single line in it:

```
$ sudo cat rvd.log.0
2017-11-27T10:52:50HKT	INFO : Main: Initializing rvd context(at rvd.c:343)
```

Can you please look into this? Thanks

**Comment by jjr840430** [comment]
fixed by #215

**Comment by ronaldtse** [comment]
Confirmed.
